client/src/dataset/addtoproject/DatasetAdd.present.js

```diff
@@ -26,16 +26,40 @@
 
 import React, { useEffect, useState } from "react";
 import { Link } from "react-router-dom";
-import { Row, Col, Modal, ModalHeader, ModalBody } from "reactstrap";
+import { Row, Col } from "reactstrap";
 import { Button } from "reactstrap";
-import { ModalFooter } from "reactstrap/lib";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faCheck, faExclamationTriangle } from "@fortawesome/free-solid-svg-icons";
 
 import { Loader } from "../../utils/components/Loader";
 import SelectAutosuggestInput from "../../utils/components/SelectAutosuggestInput";
+import { ButtonGroup, Table } from "reactstrap/lib";
+import { getDatasetAuthors } from "../DatasetFunctions";
 
-function ImportDatasetStatus(status, text, existingProject, history) {
+function HeaderAddDataset(dataset) {
```

Comment:
Why not make this a component? 
```suggestion
function HeaderAddDataset({dataset}) {
```

Reply:
no reason to don't be a component. I updated in the last commit